multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/UnbindServiceFromApplicationStep.java

```diff
@@ -23,8 +26,7 @@ protected StepPhase executeStep(ProcessContext context) {
         getStepLogger().info(Messages.UNBINDING_SERVICE_INSTANCE_FROM_APP, service, app.getName());
 
         CloudControllerClient client = context.getControllerClient();
-        client.unbindServiceInstance(app.getName(), service);
-
+        client.unbindServiceInstance(app.getName(), service, getApplicationServicesUpdateCallback(context));
```

Comment:
I don't see such method in your other PRs https://github.com/SAP/cf-java-client-sap/pull/198/files https://github.com/SAP/cf-java-client-sap/pull/199/files to be added

Reply:
The method is already existing